linux_os/guide/system/permissions/restrictions/coredumps/sysctl_fs_suid_dumpable.rule

```diff
@@ -1,7 +1,5 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,fedora
```

Comment:
This is already in Debian/ubuntu/etc: https://github.com/OpenSCAP/scap-security-guide/blob/f411d0c9394742703cd8ae68950ce964b4aff60e/debian8/guide/system/permissions/restriction/coredumps/sysctl_fs_suid_dumpable.rule

Referenced here for SUSE: https://lists.opensuse.org/opensuse-factory/2017-10/msg00171.html